zone_1:
  x: 0.0
  y: 1.308
  yaw: 3.1415
zone_2_center:
  x: -0.650
  y: 0.708
  yaw: 0.0
zone_2_to_push:
  x: -0.850
  y: 0.708
  yaw: 3.1415
zone_2_1:
  x: -0.6
  y: 1.228
  yaw: -1.5708
zone_2_2:
  x: -0.650
  y: 0.216
  yaw: 3.1415
